components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -508,21 +509,16 @@
     bind:this={main}
   >
     <div class="settings">
-      <section
-        class="basic-details"
-        class:expanded={sections[SectionNames.BASIC_DETAILS]?.expanded}
```

Comment:
Removed the concept of .expanded for now as a trial, may remove it from the object altogether if falling back on `<details>` is sufficient (read: if nobody needs a specific section to start open by default as a prop)